pkg/manager/member/template.go

```diff
@@ -236,8 +249,19 @@ then
 fi
 
 # Use HOSTNAME if POD_NAME is unset for backward compatibility.
-POD_NAME=${POD_NAME:-$HOSTNAME}
-ARGS="--pd={{ .PDAddress }} \
+POD_NAME=${POD_NAME:-$HOSTNAME}{{ if .FormatClusterDomain }}
```

Comment:
What about TiFlash, TiCDC, and Pump?

Reply:
The PD addresses in different components should all support configuring multiple addresses, should we just configure multiple addresses for them?

Reply:
why only use the new `verify` service if `.FormatClusterDomain`?